cmd/openfish/handlers/annotations.go

```diff
@@ -233,10 +233,7 @@ func CreateAnnotation(ctx *fiber.Ctx) error {
 	}
 
 	// Get logged in user.
-	observer := "testuser"
-	if ctx.Locals("email") != nil {
-		observer = ctx.Locals("email").(string)
-	}
+	observer := ctx.Locals("id").(int64)
```

Comment:
I'm probably missing it, but is this set somewhere?

Reply:
Also, is it standard practice to not check the type assertion?

Reply:
It is not set and so it won't work.  I have fixed that  now. I am now checking the type assertions.